docs/community/maintainers-guide/releasing.rst

```diff
@@ -0,0 +1,224 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2020 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Releasing
+=========
+
+In general, you have two types of releases:
+
+- **Major/Minor-level release** - Often these releases add significant new
+  features and needs to go through extensive testing.
+- **Patch-level release** - Often these releases fix security issues or
+  important bugs.
+
+For major/minor-level releases, you'll often be working only with the
```

Comment:
Minor: just cosmetic. When reading I feel that "often" is used a lot, we could exchange it with synonyms at least in this paragraph, e.g.

```suggestion
For major/minor-level releases, you'll mostly be working only with the
```